SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.Apple/Analytics.cs

```diff
@@ -64,10 +64,10 @@ internal static void Resume()
             MSACAnalytics.Resume();
         }
 
-        /// <summary>
-        /// Enable or disable automatic page tracking.
-        /// Set this to false to if you plan to call <see cref="TrackPage"/> manually.
-        /// </summary>
+        ///// <summary>
```

Comment:
?

Reply:
It was XML doc of unexisting method in compilers point of view. There is similar approach a few lines below